shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -855,14 +878,14 @@ \subsection{Block Body Transition}
 
 The Block Body Transition updates the block body state which comprises the
 ledger state and the map describing the produced blocks. The environment of the
-$\mathsf{BBODY}$ transition are the protocol parameters.
+$\mathsf{BBODY}$ transition are the genesis key delegations.
 
 \begin{figure}
   \emph{BBody Transitions}
 
   \begin{equation*}
     \_ \vdash \var{\_} \trans{bbody}{\_} \var{\_} \subseteq
-    \powerset (\PParams \times (\LState\times\BlocksMade)
+    \powerset ((\VKeyGen\mapsto\VKey) \times (\LState\times\BlocksMade)
```

Comment:
the type `VKeyGen` is not really defined anywhere

Reply:
ah yes, I meant to reference the Byron ledger spec, thanks!